sky/utils/cli_utils/status_utils.py

```diff
@@ -91,6 +90,57 @@ def show_status_table(cluster_records: List[Dict[str, Any]],
     return pending_autostop
 
 
+def show_report_table(cluster_records: List[Dict[str, Any]],
+                      show_all: bool,
+                      reserved_group_name: Optional[str] = None) -> int:
+    """Compute cluster table values and display for cost report.
+
+    Returns:
+        Number of pending auto{stop,down} clusters.
+    """
+    # TODO(zhwu): Update the information for autostop clusters.
+
+    status_columns = [
+        StatusColumn('NAME', _get_name),
+        StatusColumn('LAUNCHED', _get_launched),
+        StatusColumn('DURATION (hours)', _get_duration),
```

Comment:
Let's remove the unit from the column name and use `log_utils.readable_time_duration(0, duration, absolute=True)` for the duration